jobs/bump-lockfile.Jenkinsfile

```
@@ -298,6 +302,30 @@ lock(resource: "bump-lockfile") {
             }]}
         }
 
+        // Bump the date stamp file with today date.
+        // This will trigger a konflux build.
+        def DATE_STAMP=".tekton/trigger"
+        stage("Bump date stamp") {
+            shwrap("""
+                pushd src/config
+                today=$(date +%Y%m%d)
```

Just in case somehow the local system gets onto a non-UTC timezone let's  request UTC: 

```suggestion
                today=$(date --utc +%Y%m%d)
```